src/grafana/prometheus/Cadence-Server-Basic.json

```diff
@@ -560,17 +602,19 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "ceil(rate(cadence_requests{operation=\"RecordActivityTaskStarted\"}[10s]))",
+          "exemplar": true,
+          "expr": "(1-sum(cadence_errors{cadence_service=\"cadence_frontend\",namespace=\"$namespace\"})/sum(cadence_requests{cadence_service=\"cadence_frontend\",namespace=\"$namespace\"}))*100",
```

Comment:
I think this is still server metrics. If it's difficult to get client metrics here, we can skip this panel.